.gitlab-ci.yml

```diff
@@ -125,6 +125,11 @@ package:windows-amd64:
    tags: [go]
    script: go run mage.go -v PackageWindowsAmd64
 
+package:ios:
```

Comment:
I think you can use the tag ios instead of packageIOS, it becomes a standard tag from 1.16 onwards.